Cards/Movement State/MoveState_Discarding.gd

```diff
@@ -0,0 +1,18 @@
+extends CardMovementState
+class_name MoveState_Discarding
+
+
+# @Override
+func on_state_enter() -> void:
+	# Move to discard pile
+	_state.card.create_tween()\
```

Comment:
was this function `create_tween` function declared previously ? if not what does it do ?
based on the name I would guess it's something for curves maybe ?

Reply:
`create_tween` is a built in godot function. It pretty much creates a request to change a value over time.  [https://docs.godotengine.org/en/stable/classes/class_tween.html](url)